Tests/SentryTests/Integrations/SentrySessionTrackerTests.swift

```diff
@@ -28,15 +28,22 @@ class SentrySessionTrackerTests: XCTestCase {
     }
     
     private var fixture: Fixture!
+    private var sut: SessionTracker!
 
     override func setUp() {
         super.setUp()
 
         fixture = Fixture()
```

Comment:
L: `Fixture` should not have a state, so it's not actually needed to create an instance every time.

Reply:
I'm also a fan of avoiding states when possible. In order to validate my tests, I need a new `fileManager` for each running test. This is some type of state I need to keep somewhere for every test. Where would you put it then?

Reply:
@philipphofmann the `Fixture` is an internal class for getting a new `sut` with all its dependencies.
you need a new `sut` on each test, but not a new `Fixture`.
The `Fixture` has only immutable fields, and the `getSut()` method may create new instances of mutable params.

Reply:
Thanks for explaining the pattern. Where would you keep the stateful `fileManager` then?